src/Faction.js

```diff
@@ -5,15 +5,35 @@ class Faction {
   }
 
   static fromCode (code) {
-    if (!Faction.FACTIONS.includes(code)) throw new TypeError('Invalid faction code!')
-    return new this(code, this.FACTIONS.indexOf(code))
+    const factionId = Faction.FACTIONS[code]
+
+    if (factionId === undefined) {
+      throw new TypeError('Invalid faction code. It is possible you need to upgrade the runeterra package.')
```

Comment:
I updated the message here so that the user is signaled to update this package. Wording can be modified if wanted.